environment.yml

```diff
@@ -2,21 +2,31 @@ name: geocat-applications
 channels:
   - conda-forge
 dependencies:
+  # Content
   - cartopy
   - geocat-datafiles
   - matplotlib
   - netcdf4
   - numpy
-  - pre-commit
+  - pandas
   - python
   - xarray
+
+  # Jupyter
   - ipykernel
   - ipython
   - jupyter_client
-  - matplotlib-base
-  - sphinx-book-theme
+
+  # Infrastructure
   - myst-nb
+  - nbsphinx
+  - pre-commit
+  - sphinx-book-theme
   - sphinx-design
   - sphinxcontrib-bibtex
-  - nbsphinx
-  - pandas
+
+  # pip
+  - pip
+  - pip:
+    - pythia-datasets # content
+    - nc-time-axis # content
```

Comment:
Both of these are on conda-forge as well.  Fine either way though.

Reply:
Should `Jupyter` include `jupyterlab`? I use the `jupyter lab` command to run jupyter locally for dev

Reply:
I did not know this, I just assumed they were pip only because both only had pip download instructions. I'll go switch them to conda, thanks for noticing this!!